### PR TITLE
Handle `osu_user.previous_usernames` = None

### PR DESCRIPTION
**Pull request by FoxeiZ**
**Describe your PR**
Add a checker for `osu_user.previous_usernames` in case it returns None

**What does it do**
Check the value of `previous_usernames` before calling `add_field`

**Did you test it locally?**
*hehe. no*

### CODE REVIEW

File: database/crud.py [hunk 1]
```diff
@@ -24,9 +24,7 @@ def __init__(self):
 
         if self.is_mongo:
             self.mongo_client = pymongo.MongoClient(Utility.get_db_url())
-            self.mongo_engine: Database = self.mongo_client[
-                Config.DATABASE["db_name"]
-            ]
+            self.mongo_engine: Database = self.mongo_client[Config.DATABASE["db_name"]]
```
Comment (Swyreee) [inline database/crud.py:27]: kekw

Comment (FoxeiZ) [reply database/crud.py:27]: black format đấy

File: cogs/OsuCog.py [hunk 2]
```diff
@@ -181,6 +177,12 @@ async def __generic_check(
                 )
             )
 
+            if osu_user.previous_usernames:
+                eb.add_field(
+                    name="Formerly known as",
+                    value=", ".join(osu_user.previous_usernames),
+                )
+
```
Comment (Swyreee) [inline cogs/OsuCog.py:185]: có gì sử dụng [`Embed.insert_field_at()`](https://discordpy.readthedocs.io/en/master/api.html?highlight=embed#discord.Embed.insert_field_at) đi :eyes:, nhét vào index `0` á

Comment (FoxeiZ) [reply cogs/OsuCog.py:185]: sure để commit

**Pull request merged**